config/action.d/pf.conf

```diff
@@ -16,7 +16,7 @@
 # we don't enable PF automatically; to enable run pfctl -e 
 # or add `pf_enable="YES"` to /etc/rc.conf (tested on FreeBSD)
 actionstart = echo "table <<tablename>-<name>> persist counters" | pfctl -f- 
-              echo "block proto <protocol> from <<tablename>-<name>> to any port <port>" | pfctl -f-
+              echo "block proto <protocol> from <<tablename>-<name>> to any<actiontype>" | pfctl -f-
```

Comment:
>  > -              echo "block proto <protocol> from <<tablename>-<name>> to any port <port>" | pfctl -f-
>  > +              echo "block proto <protocol> from <<tablename>-<name>> to any<actiontype>" | pfctl -f-
>      Since we could also simply just strip lines on the right I guess before
>      execution...
>    Impossible here, because after the space we've piping... But the test
>    cases are exact :)

d'oh -- right!   but then here it is ok to just leave space after any
and have "any <actiontype>" without requiring any specific leading
"<sp>" in actiontype, which could easily be omitted by users in their
"customizations" etc.  I am just trying to get it more obvious -- you
already saw how ppl can shoot them in the foot with even simpler more
obvious cases ;)